bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -201,6 +205,19 @@ func Run(opt *options.ServerOption) error {
 		return err
 	}
 
+	log.Infof("create rolebinding kubeflow-admin for role permission")
```

Comment:
Any particular reason we're shelling out to kubectl rather than using the goclient to make a request to the API?